source/gui/settingsDialogs.py

```diff
@@ -1703,6 +1702,19 @@ def makeSettings(self, settingsSizer):
 		self.autoDialectSwitchingCheckbox.SetValue(
 			config.conf["speech"]["autoDialectSwitching"],
 		)
+		# Translators: This is the label for a checkbox in the
+		# voice settings panel (if checked, the language switched to will be reported).
```

Comment:
Why a so early carriage return? And why is the description between brackets?

Reply:
I copied some piece of code comment, but now I've fixed it. I didn't know if adding a longer line would cause linting errors.